ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/UPLINK_P2P_VRFS_RFC5549_TESTS_SPINE1.cfg

```diff
@@ -0,0 +1,60 @@
+!
+no enable password
+no aaa root
+!
+vlan internal order ascending range 1006 1199
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname UPLINK_P2P_VRFS_RFC5549_TESTS_SPINE1
+!
+spanning-tree mode none
+!
+vrf instance MGMT
+!
+management api http-commands
+   protocol https
+   no shutdown
+   !
+   vrf MGMT
+      no shutdown
+!
+interface Ethernet3
```

Comment:
I would have expected a subinterface created for the VRF IT here the way p2p-vrfs works (and actually some for the PROD one as well)

can you investigate why they are not created? 
Thanks!